### DISCUSSION TITLE
Power Button shows "There are currently no power options available"

### DISCUSSION THREAD

**Discussion by Velocity-01** [SafeExamBrowser/seb-win-refactoring#1000] [Q&A] [answered]
When i try to shutdown my pc the power Button shows "There are currently no power options available". I have tried restoring the power plan by using Command Prompt but that still did not fix it. I cannot shutdown or restart my pc with the power button so now i need to use command prompt and type "shutdown /s". This issue occured after i had to use Safe Exam Browser but have used it multiple times in the past and had no issues until now. Is there anything i can do to fix this issue?

![Screenshot 2024-10-16 183907](https://github.com/user-attachments/assets/e92f3fef-9dea-43db-9f82-99c87a153f18)

**Comment by dbuechel** [comment, marked as answer]
Which version are / were you using? Please use the Reset Utility which is part of every standard SEB installation to restore or reset any changes SEB made to the system configuration.

**Comment by Velocity-01** [comment]
I just used the Reset Utility and it worked. I did not know you could do this. Thank you very much!
